
C:\src\wxCVS\src\common>rem set CVSROOT=:pserver:anonymous@cvs.wxwindows.sourceforge.net:/cvsroot/wxwindows 

C:\src\wxCVS\src\common>rem set USERNAME=anonymous 

C:\src\wxCVS\src\common>rem set HOMEDRIVE=c: 

C:\src\wxCVS\src\common>rem set HOME=\src 

C:\src\wxCVS\src\common>rem set HOMEPATH=\src 

C:\src\wxCVS\src\common>set CVSROOT=:pserver:anoncvs@cvs.wxwindows.org:/home/wxcvs 

C:\src\wxCVS\src\common>set USERNAME=anoncvs 

C:\src\wxCVS\src\common>set HOMEDRIVE=c: 

C:\src\wxCVS\src\common>set HOME=\src 

C:\src\wxCVS\src\common>set HOMEPATH=\src 

C:\src\wxCVS\src\common>cvs -Q diff -u object.cpp      
Index: object.cpp
===================================================================
RCS file: /home/wxcvs/wxWindows/src/common/object.cpp,v
retrieving revision 1.60
diff -u -r1.60 object.cpp
--- object.cpp	6 Feb 2002 01:38:48 -0000	1.60
+++ object.cpp	1 Apr 2002 23:44:10 -0000
@@ -161,7 +161,7 @@
     wxClassInfo *classInfo = (wxClassInfo *)sm_classTable->Get(name);
 
     // this must be fixed, other things risk work wrongly later if you get this
-    wxASSERT_MSG( classInfo, _T("base class unknown to wxWindows RTTI") );
+    wxASSERT_MSG( classInfo, wxString(_T("base class ")) + name + _T(" unknown to wxWindows RTTI") );
 
     return classInfo;
 }

 	  	 
